matchbatch.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
I'd kind of prefer to see all ruby in this repo, since we're already using it for ticketmatch proper, but I am glad that this agent-specific stuff is separated from the main script, and this looks useful. Whatever works.

Reply:
Thats the medium term plan.